### DISCUSSION TITLE
ancombc2 function

### DISCUSSION THREAD

**Discussion by madbullahi** [FrederickHuangLin/ANCOMBC#158] [General]
I tried to run the ancombc2 function but it doesn't pop up when i load the ancombc package.
Please help
<img width="960" alt="2023-03-15" src="https://user-images.githubusercontent.com/94472416/225288142-25f53187-d8b5-4a72-bff5-8db4ed667ad8.png">

**Comment by RachBioHaz** [comment]
How did you install/load the package? If you can provide the information we can identify if it's a basic issue like it's not loading properly or if there is some other issue.  I can only see part of your library. Do a search in the Packages search box to see if it's installed at all, then check if it is selected (loaded). If neither has happened, try the installation process again. Then if that doesn't work update this thread with what you've tried.
Cheers
